src/hooks/useCrossHair.tsx

```diff
@@ -0,0 +1,66 @@
+import { CSSProperties, useState } from 'react';
+
+import { Group } from '../components/Annotations/Group';
+import { AnnotationLineProps, Line } from '../components/Annotations/Line';
+import { Text } from '../components/Annotations/Text';
+import { usePlotEvents } from '../contexts/plotController/plotControllerContext';
+
+export interface UseCrossHairOptions {
+  axisId?: { x: string; y: string };
```

Comment:
Use these two props like in the rectangular zoom hook: https://github.com/zakodium/react-plot/blob/d86e2e2f11f290f0ce2b8a4842ce16e6e17811e9/src/hooks/types.ts#L3-L6

Reply:
fixed